packages/superset-ui-legacy-preset-chart-deckgl/src/DeckGLContainer.jsx

```diff
@@ -24,6 +24,8 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { StaticMap } from 'react-map-gl';
 import DeckGL from 'deck.gl';
+// eslint-disable-next-line import/extensions
+import Tooltip from './components/Tooltip';
```

Comment:
any reason to not just add `.tsx`? (def a style thing 🤷‍♂ )

Reply:
This will cause file not found issue after transpile because the code will remain 

```ts
import Tooltip from './components/Tooltip.tsx';
```

but `Tooltip.tsx` also became `Tooltip.js` through transpilation already.